coreapi/main_api.py

```diff
@@ -33,6 +38,154 @@
 logger = RekogntionLogger(name="main_api")
 
 
+def text_reco(image):
+    """     Scene Text Recognition
+    Args:
+            *   image: numpy array of cropped text
+    Workflow:
+            *   A numpy array of a cropped text is taken as input
+                inference input dimension requires dimension of (100,32)
+                therefore the input is first resizing to required
+                input dimension and then normalized.
+            *   Now the processed output is further processed to make it a
+                json format which is compatible to TensorFlow Serving input.
+            *   Then a http post request is made at localhost:8501.
+                The post request contain data and headers.
+            *   Incase of any exception, it return relevant error message.
+            *   output from TensorFlow Serving is further processed using
+                wordninja and then returned as a dictionary which keeps Text
+                as key and processed output as value.
+    Returns:
+            *   Dictionary having text as Key and processed output as value.
+    """
+
+    logger.info(msg="text_reco called")
+    image = cv2.resize(image, tuple((100, 32)), interpolation=cv2.INTER_LINEAR)
+    image = np.array(image, np.float32) / 127.5 - 1.0
+    data = json.dumps({"inputs": [image.tolist()]})
+    try:
```

Comment:
line no from  66-88 has been repeated many times in this project, can we avoid this?

Reply:
sure can make a function in utils.py for that

Reply:
will do in next PR from refactor branch